README.md

```diff
@@ -1,2 +1,148 @@
-# bookstore
-This site displays a bookstore where you can add and remove books from the collection. Created using React and Redux.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>MO Bookstore</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 MO Bookstore <a name="about-project"></a>
+
+**MO Bookstore** is a simple website to save and display a list of books. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React JS</li>
+    <li>Webpack</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Save and display unlimited amounts of books]**
+- **[Display author and title]**
+- **[Add and delete functions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this [repository](https://github.com/maov19/bookstore) to your desired folder.
+
+### Install
+
+No installation required.
+
+### Usage
+
+To run the project:
+    - Run `npm install`
+    - Run `npm run start`
+
+### Run tests
+
+To run tests:
+    - Run `npm run test`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Martin Ossa**
+- GitHub: [@maov19]
+
```

Comment:
- [ ] You have done an excellent job implementing the features for this project. I would kindly suggest you add the correct GitHub and LinkedIn links to your README document and ensure it is clickable.

Reply:
thanks @Yerimah! the requested changes were already added 🔧